src/vs/base/common/codiconsLibrary.ts

```diff
@@ -318,7 +318,7 @@ export const codiconsLibrary = {
 	repoPush: register('repo-push', 0xeb41),
 	report: register('report', 0xeb42),
 	requestChanges: register('request-changes', 0xeb43),
-	rerunTask: register('rerun-task', 0xead2),
+	rerunTask: register('rerun-task', 0xeb37),
```

Comment:
@meganrogge you should remove this line and use Codicon.refresh in the tasks folder